mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -618,41 +606,45 @@ def quantized_values(self) -> Optional[Union[Iterable[int], Iterable[float]]]:
         """
         num_range = self.range
         if self.type == "float":
-            if not self._quantization:
+            if not self.cardinality:
```

Comment:
This looks circular to me.  cardinality reads off of quantization and vice-versa?

Reply:
Ah, nm, this is quantized values.  The diff got folded up in a confusing way.